### PR TITLE
Feature/bugfixes manager

### PR DESCRIPTION
**Pull request by mirakst**
Lots of changes - notable ones include, but are not limited to:

* Redesigned detached manager window - now shows more info in detailed view, switches sides  when 'expanded'.

* Search is now fuzzy search.

* Can move managers with double click or enter, added tooltips to describe actions on managers in datagrids, or their name in the case of the chart.

* Improved manager charts vastly, now only plots if they have 'enough' data, view is fitted to series, can zoom.

* Changed cursor for all plots, added wait cursor when main window loading. 

* Manager can now be inspected in detached window by double clicking in main window manager view

### CODE REVIEW

File: DashboardFrontend/DetachedWindows/LogDetached.xaml [hunk 1]
```diff
@@ -194,6 +194,7 @@
                             <Setter Property="FontSize" Value="14"/>
                             <Setter Property="Focusable" Value="False"/>
                             <Setter Property="ToolTip" Value="{Binding ManagerName, TargetNullValue='Context not found'}"/>
+                            <Setter Property="ToolTipService.InitialShowDelay" Value="200"/>
```
Comment (damniko) [inline DashboardFrontend/DetachedWindows/LogDetached.xaml:197]: nice

File: DashboardFrontend/DetachedWindows/ManagerListDetached.xaml.cs [hunk 2]
```diff
@@ -30,13 +31,18 @@ public partial class ManagerListDetached : Window
         /// <param name="e"></param>
         private void AddManager_Click(object sender, RoutedEventArgs e)
         {
+            List<ManagerWrapper> managers = new();
             foreach (ManagerWrapper manager in DatagridManagers.SelectedItems)
             {
-                if (!Vm.WrappedManagers.Any(e => e.Manager.ContextId == manager.Manager.ContextId))
+                if (!Vm.DetailedManagers.Any(m => m.Manager.ContextId == manager.Manager.ContextId))
```
Comment (damniko) [inline DashboardFrontend/DetachedWindows/ManagerListDetached.xaml.cs]: Vil det her stadig virke når alle managers har samme context ID (0) ?

File: DashboardFrontend/ViewModels/ExecutionObservable.cs [hunk 3]
```diff
@@ -13,7 +13,9 @@ public ExecutionObservable(Execution exec)
         {
             Id = exec.Id;
             StartTime = exec.StartTime;
+
             Managers = new(exec.Managers.Select(m => new ManagerObservable(m)));
```
Comment (damniko) [inline DashboardFrontend/ViewModels/ExecutionObservable.cs]: ingen for-loop eller locking?

File: DashboardFrontend/ViewModels/LogViewModel.cs [hunk 4]
```diff
@@ -174,6 +174,7 @@ public bool ShowValidation
         public void UpdateData(List<Execution> executions)
         {
             Executions = new(executions.Select(e => new ExecutionObservable(e)));
```
Comment (damniko) [inline DashboardFrontend/ViewModels/LogViewModel.cs]: und hier?

File: DashboardFrontend/ViewModels/MainWindowViewModel.cs [hunk 5]
```diff
@@ -52,6 +53,8 @@ public Visibility LoadingVisibility
                 OnPropertyChanged(nameof(LoadingVisibility));
             }
         }
+
+        public Window Window { get; set; }
```
Comment (damniko) [inline DashboardFrontend/ViewModels/MainWindowViewModel.cs]: well nu hvor jeg kigger på det ser det sku mærkeligt ud at flytte logikken i en anden klasse, når man alligevel er nødt til at bruge selve vindue-klassen... MEN så længe det virker synes jeg bare vi skal feje det under gulvtæppet

File: Model/Manager.cs [hunk 6]
```diff
@@ -34,15 +34,7 @@ public string Name
...
-                var splitName = value.Split('.');
-                if (splitName.Contains("managers"))
-                {
-                    ShortName = "(...)" + string.Join(".", splitName.TakeLast(2));
-                }
-                else
-                {
-                    ShortName = "(...)" + string.Join(".", splitName.Skip(4));
-                }
+                ShortName = value.Split(".").Last();
```
Comment (damniko) [inline Model/Manager.cs:37]: awesome

**Review by damniko — APPROVED** [review]
very nice

File: DashboardFrontend/DetachedWindows/ManagerListDetached.xaml.cs (hunk 2 above)
Comment (mirakst) [reply DashboardFrontend/DetachedWindows/ManagerListDetached.xaml.cs]: no

File: DashboardFrontend/ViewModels/MainWindowViewModel.cs (hunk 5 above)
Comment (mirakst) [reply DashboardFrontend/ViewModels/MainWindowViewModel.cs]: jeg bruger ikke længere mainwindow 👍

File: DashboardFrontend/ViewModels/LogViewModel.cs (hunk 4 above)
Comment (mirakst) [reply DashboardFrontend/ViewModels/LogViewModel.cs]: ordnet

File: DashboardFrontend/ViewModels/ExecutionObservable.cs (hunk 3 above)
Comment (mirakst) [reply DashboardFrontend/ViewModels/ExecutionObservable.cs]: ordnet

File: DashboardFrontend/DetachedWindows/ManagerListDetached.xaml.cs (hunk 2 above)
Comment (mirakst) [reply DashboardFrontend/DetachedWindows/ManagerListDetached.xaml.cs]: no, ændret til navn

File: DashboardFrontend/DetachedWindows/LogDetached.xaml (hunk 1 above)
Comment (mirakst) [reply DashboardFrontend/DetachedWindows/LogDetached.xaml:197]: du

File: Model/Manager.cs (hunk 6 above)
Comment (mirakst) [reply Model/Manager.cs:37]: yessirz

**Review by Mthage20 — APPROVED** [review]

**Pull request merged**